packages/expo-media-library/android/src/main/java/expo/modules/medialibrary/assets/CreateAsset.kt

```diff
@@ -17,16 +18,23 @@ import expo.modules.medialibrary.ERROR_IO_EXCEPTION
 import expo.modules.medialibrary.ERROR_UNABLE_TO_LOAD_PERMISSION
 import expo.modules.medialibrary.ERROR_UNABLE_TO_SAVE
 import expo.modules.medialibrary.MediaLibraryUtils
+import expo.modules.medialibrary.albums.getAlbumFileOrNull
 import java.io.File
 import java.io.FileInputStream
 import java.io.FileOutputStream
 import java.io.IOException
 
-class CreateAsset @JvmOverloads constructor(
+/**
+ * Creates an asset entry in the provided albumFile.
+ * If no file has been provided the asset will be created in the default directory for the mimeType
+ * of the file under the `uri` parameter.
+ */
+class CreateAssetWithAlbumFile(
```

Comment:
Splitting `CreateAsset` into two functions may seem a bit unnecessary, but it becomes more useful in the [2/2] PR